content/04kapitel.tex

```diff
@@ -273,6 +273,82 @@ \subsubsection{SKIOS-64 Guideline for Issues}
 Bearbeitet von Simon Morgenstern.
 Reviewed von Amos Groß.
 
+\subsubsection{SKIOS-11 Update Mockups for possible design language}
+Das Ticket \enquote{SKIOS-11 Update Mockups for possible design language} hatte folgende User Story:
+\begin{quotation}
+    As a ui/ux designer, I would like to like to have a design language to follow when creating mockups.
+
+    If you’re unsure about what a design langage is, the following article explains it:  (this story references the “Style guidelines” part of their definition)
+
+    Also interesting thing to look up might be UI/UX design in general to understand why design languages matter and are necessary for good software.
+\textbf{Acceptance Criteria}
+\begin{itemize}
+    \item deas are documented in confluence
+    \item Figma Mock ups now represent new design language
+    \item Decisions on fonts, separation of pages and page structure are made
+    \item A clear design language is implemented for Skiosa
+\end{itemize}
+\end{quotation}
+Bearbeitet von Amos Groß.
+Reviewed von Lukas Huida.
+
+\subsubsection{SKIOS-27 Document Initial Requirements}
+Das Ticket \enquote{SKIOS-27 Document Initial Requirements} hatte folgende User Story:
+\begin{quotation}
+    As a product owner I want the plattform's requirements and features to be documented, so that we can refer to it in the final documentation.
```

Comment:
```suggestion
    As a product owner I want the plattform's requirements and features to be documented, so that we can refer to it in the final documentation.
```
```suggestion
    As a product owner I want the plattform's requirements and features to be documented, so that we can refer to it in the final documentation. \\
```